tickets/settings/local.py

```diff
@@ -8,7 +8,11 @@
 
 DATABASES = {
```

Comment:
Please also add to prod settings, take into account that for prod code should read envs from .env